### DISCUSSION TITLE
Home Assistant Glow - Case - ESP32 Top Side - Prone to splitting

### DISCUSSION THREAD

**Discussion by YorkshireHillbilly** [klaasnicolaas/home-assistant-glow#235] [Ideas]
First of all... GREAT WORK!

Has anyone else found the ESP32 Top Side is prone to splitting when trying to push the ESP32 Bottom Side into it?

Would it be possible to have a bridge put in after the last used pins to strengthen the side walls and stop them from splitting. I've busted 2 so far.

![image](https://user-images.githubusercontent.com/1895882/204295197-b1c9757c-2ce7-4f4c-8c04-9ed6df1992fd.png)

*this is my 1st 3D printing project, I'm nowhere the level of editing existing objects yet.

**Comment by KixMan28** [comment]
Hi @YorkshireHillbilly What material have you used to print the case? I have printed several cases both in PLA and PETG and none of the broke. If you make a bridge at the location you marked with red you won't be able to slide the top of the case on the bottom of the case.
Can you share a picture with your broken case with the ESP32 inside?

**Reply by YorkshireHillbilly** [reply]
PLA

![image](https://user-images.githubusercontent.com/1895882/204337059-3f659eb0-8245-4e9c-8733-777514e0da4e.png)

Maybe I didn't print them properly. I'll go looking in the settings / preferences to see if there's any kind of Low / Medium / High or Proof / Best options.

**Reply by YorkshireHillbilly** [reply]
I found an option to make the print "Hard" so I selected that. The print felt sturdier however, when I inserted the bottom side, the same piece snapped off in the same place.

Could the 2 'connecters' at the end of the bottom side be shortened (as shown below) to allow for 2 supporting pieces to be introduced on the top side (as shown above)?

![image](https://user-images.githubusercontent.com/1895882/204513875-bd001bbb-67f5-43da-8f52-fe048de9ab8c.png)

**Reply by KixMan28** [reply]
In my opinion your top part printed a little bit scaled down, and that is causing the side to snap off. As I told you in my previous message I printed several cases and non of the snapped of. Actually I felt them a little bit loose.

Are you using the ESP32 case STL files found here, linked to this project?

Which slicer do you use? I don't remember any setting for "hard" printing. I use Cura and PrusaSlicer.
Which printer brand and model do you use?

**Reply by KixMan28** [reply]
I just checked the STL files uploaded here. I hope you didn't print the objects in the orientation they are in the files? You should print them in horizontal orientation with the biggest surface touching your print bed, the way they are positioned in your picture above.
How many walls/outer shells are set in your slicer? 
What percentage infill do you use?
Which infill type do you use?

**Reply by klaasnicolaas** [reply]
I have never experienced any problems printing the cases, I print the top part of the ESP case with the long side up. You also have a handy plugin in Cura that can automatically put your STL in the best position for printing.

![image](https://user-images.githubusercontent.com/20448157/204891881-d10c9d9c-ebcb-4d1d-ab40-43248429b8f3.png)
